common/supplemental/supplementalData.xml

```diff
@@ -1734,7 +1734,7 @@ XXX Code for transations where no currency is involved
 		<language type="kg" scripts="Latn"/>
 		<language type="kg" territories="CD" alt="secondary"/>
 		<language type="kge" scripts="Latn"/>
-		<language type="kgp" scripts="Latn"/>
+		<language type="kgp" scripts="Latn" territories="BR"/>
```

Comment:
This data will be overwritten when the tools are run. See line 1270. I believe that the tools will add the territories that are in common/main, but that needs to be checked.

Reply:
I've checked instructions available on 1270. When we run `ConvertLanguageData.java`, three files are generated (country_language_population_raw.txt, language_script_raw.txt, supplementalData.xml). On `supplementalData.xml`, the territories are not added because the new languages do not meet the existing rules (major/official status, at least 20% of whole population, or 1M), so it's not considered as primary/secondary.

Is **ConvertLanguageData** being used to generated supplementalData.xml?

Maybe we should remove this, but want to hear from you.